src/app/components/annotations/components/grid-tile-content/grid-tile-content.component.html

```diff
@@ -0,0 +1,44 @@
+<div #wrapper class="content-wrapper">
+  <a [href]="listenLink" class="content-link">Go To Source</a>
+
+  <button
+    (click)="toggleContext()"
+    class="content-link btn btn-sm btn-outline-primary context-anchor"
+    aria-label="Show additional context around audio event"
+  >
+    Show More
+  </button>
+</div>
+
+@if (contextExpanded) {
+<div class="context-card context-anchor-item">
+  <div class="context-card-header">
+    <span class="context-card-information">
+      <span class="context-card-title">Context</span>
+
+      <span class="context-card-description ms-2">(Showing 30 seconds around audio event)</span>
+    </span>
+
+    <span class="context-card-actions">
+      <button (click)="contextExpanded = false" class="btn btn-outline-danger">
+        Close
+      </button>
+    </span>
+  </div>
+
+  <div class="context-card-elements">
+    <oe-axes>
+      <oe-indicator>
+        <oe-spectrogram
+          [id]="contextSpectrogramId"
+          [src]="contextSource"
+          color-map="audacity"
```

Comment:
we should use grayscale here to match our host site

todo later: spectrogram settings per user site wide